### ISSUE TITLE
Bug: In rare occasions, local tags cannot get attached to attributes

### ISSUE THREAD

**Issue by 3c7** [MISP/MISP#9451]
### Actual behavior

In rare occasions, adding a local tag to an attribute results in either:

```json
{"name":"Invalid attribute","message":"Invalid attribute","url":"\/attributes\/addTag"}
```

or

```json
{"saved":false,"errors":"Tag could not be added."}
```

In both cases

- the attribute is available
- the attribute has correlating attributes with the same tag
- adding the tag via GUI works

### Expected behavior

Adding the local tag via API should work.

### Steps to reproduce

Unsure about how to reproduce it. Creating multiple events with correlating attributes and try to add local tags via REST API.

### Version

2.4.178

### Operating System

Ubuntu

### Operating System version

20.04

### PHP version

7.4

### Relevant log output

```shell
Unfortunately, the error is visible in logs.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 3c7** [comment]
Was able to reproduce it:

- Create two events with the same attribute, e.g. `ip-src|1.1.1.1`
- Add the local tag to one of them
- Try to add the local tag via `POST /attributes/addTag` with `{"attribute": "<attribute uuid>", "tag": "<tag id>", "local": 1}`

**Comment by 3c7** [comment]
This is in issue with UUIDs being casted to integers in MySQL, according to @iglocska.

**Comment by iglocska** [comment]
Implemented a catcher for UUIDs on the current HEAD, just make sure you update if you want to use uuids instead of IDs.

**Comment by adulau** [comment]
Potential fixed in 8b4bb3b34abf6caf72184e6ff3420ad796ec2533 - feel free to reopen if it's not fixed.
